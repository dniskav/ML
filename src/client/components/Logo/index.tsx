import React from 'react';
import styled from 'styled-components';
import logo from '../../assets/Logo_ML.png';


const StyledLogo = styled.img`
  width: 53px;
  height: 36px;
`;

const Logo: React.FC = () => {

  return(
    <StyledLogo src={logo} onClick={() => console.log('logo clicked!')}/>
  )
};

export default Logo;
