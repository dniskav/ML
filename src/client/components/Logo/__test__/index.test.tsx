import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import Logo from '../index';

jest.mock('react-redux');

describe('<Logo />', () => {
  it("render success", () => {
    const wrapper: ShallowWrapper = shallow(<Logo />);
    expect(wrapper).toMatchSnapshot();
  });
});
