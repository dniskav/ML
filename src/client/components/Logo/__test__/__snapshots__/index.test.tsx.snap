// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> render success 1`] = `ShallowWrapper {}`;
